src/js/pyodide.ts

```diff
@@ -319,7 +319,10 @@ export async function loadPyodide(
     throw new Error("Lock file version doesn't match Pyodide version");
   }
   if (config.fullStdLib) {
-    await pyodide.loadPackage(["distutils"]);
+    let unvendoredStdlibs = Array.from(
+      API._pyodide._importhook.UNVENDORED_STDLIBS
+    );
+    await pyodide.loadPackage(unvendoredStdlibs);
```

Comment:
`pyodide.loadPackage` accepts a `pyproxy`:
https://github.com/pyodide/pyodide/blob/main/src/js/load-package.ts#L356
```suggestion
    await pyodide.loadPackage(API._pyodide._importhook.UNVENDORED_STDLIBS);
```

Reply:
Thanks!